changelog/12485.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+http: Enable users to customize HTTP response headers
```

Comment:
The format for a `feature` entry should be as follows:

```
**Customizable HTTP Headers**: Add support to define custom HTTP headers for root path (`/`) and also on API endpoints (`/v1/*`)
```

The description can be whatever you want, I just wanted to put something there that was related to the PR.

Reply:
hmm, seems like there are various formats for a `feature`. I am following this and not see the suggested format
https://github.com/hashicorp/vault/blob/main/CHANGELOG.md
though, I do see the suggested format in the changelog directory.

Reply:
I don't think we have been consistent but the "Key Management Secrets Engine (Enterprise)" line in the unreleased 1.9 section aligns with what I mentioned. I commented on it because I was informed that used the incorrect format as well for the "secrets/kv: Add ability to specify version-agnostic custom key metadata" line awhile after it had been merged.

Reply:
Sounds good! Thanks Chris!

Reply:
The feature section of the changelog is our chance to do a fancier formatting and more high-level description of major features in the new major release (see https://github.com/hashicorp/vault/blob/main/CHANGELOG.md#140-april-7th-2020 for an example).

Reply:
Thanks, Done